Week1/homework/Exercise4.js

```diff
@@ -0,0 +1,10 @@
+//Exercise 4
+
+'use strict';
+
+let myString = 'Hanbing Li';
+console.log('its value should be \"Hanbing Li\"');
+console.log(myString);
+myString = "it's a new string";
+console.log("its value should be 'it's a new string'");
+console.log(myString);
```

Comment:
Are you mixing ' and " to try it out? That's o.k.!

In a real world program though you should try to be consistent and use the same quotestyle to start and end all your strings.